Grid docs: Update ColumnStates section (#758)
telerik/blazor-docs: components/grid/state.md

### Get Current Columns Visibility, Order, Field

> The `ColumnStates` field of the state object provides you with information about the current state of the grid columns. It contains the following fields:

Reviewer: Fields are normally readonly, so this is a property.

```suggestion
The `ColumnStates` property of the GridState object provides you with information about the current state of the Grid columns. It contains the following properties:
```

Field | Type | Description
---------|----------|---------
 `Index` | `int` | the current index of the column based on the position the user chose

>  `Id` | `string` | the Id of the column if it has one set

Reviewer: ```suggestion
 `Id` | `string` | the Id of the column if it is set
```

 `Field` | `string` | the field of the column
 `Visible` | `bool?` | whether the column is hidden or not
 `Locked` | `bool` | whether the column is locked or not

>  `Width` | `string` | the width of the column if it has one set

Reviewer: ```suggestion
 `Width` | `string` | the width of the column if it is set
```
